pkg/releaser/releaser.go

```diff
@@ -326,7 +326,12 @@ func (r *Releaser) CreateReleases() error {
 			asset := &github.Asset{Path: provFile}
 			release.Assets = append(release.Assets, asset)
 		}
-
+		if r.config.SkipExisting {
+			existingRelease, _ := r.github.GetRelease(context.TODO(), releaseName)
+			if existingRelease != nil && len(existingRelease.Assets) > 0 {
```

Comment:
I wonder if `len(existingRelease.Assets) > 0` makes sense. I'd tend to remove that.

Reply:
@unguiculus thank you! removed the check